site/components/src/main/java/uk/nhs/hee/web/components/CookiesBannerComponent.java

```diff
@@ -0,0 +1,38 @@
+package uk.nhs.hee.web.components;
+
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.component.HstResponse;
+import org.hippoecm.hst.core.parameters.ParametersInfo;
+import org.onehippo.cms7.essentials.components.EssentialsDocumentComponent;
+import org.onehippo.cms7.essentials.components.info.EssentialsDocumentComponentInfo;
+
+import javax.servlet.http.Cookie;
+
+/**
+ * Component class for {@code cookies} abstract base component
+ */
+@ParametersInfo(type = EssentialsDocumentComponentInfo.class)
+public class CookiesBannerComponent extends EssentialsDocumentComponent {
```

Comment:
Since, `CookiesBannerComponent ` class isn't an `EssentialsDocumentComponent` based component, we can probably extend it directly from `org.onehippo.cms7.essentials.components.CommonComponent`. Also, we may not need ParametersInfo annotation `@ParametersInfo(type = EssentialsDocumentComponentInfo.class)`.

Reply:
Done.